src/index.js

```diff
@@ -2,16 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
```

Comment:
- [ ] In your project, there is no file named "index.css." The app shows an error when this line is present, so kindly remove it.

Reply:
oh it seems i forgot to remove this 
its okay i will remove it